spec/reset_password_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'password reset', type: :feature, js: true do
```

Comment:
would it make sense to put this in a `features` subdir in `spec` so that it's more obvious when browsing that it's a feature spec?